public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -99,12 +109,13 @@ export const AlertRuleForm: FC<Props> = ({ existing, prefill }) => {
     return {
       ...getDefaultFormValues(),
       queries: getDefaultQueries(),
+      recordingRuleQueries: getDefaultRecordingRulesQueries(rulesSourcesWithRuler),
```

Comment:
why we don't use the queries field ?

Reply:
Great suggestion! Just pushed the changes we tried out in our 🍐-ing session